src/ui/profile_preferences.ui

```diff
@@ -166,25 +166,70 @@
         </widget>
        </item>
        <item row="1" column="0">
-        <widget class="QGroupBox" name="groupBox_encoding">
-         <property name="enabled">
-          <bool>true</bool>
+        <widget class="QGroupBox" name="groupBox_guiLanguage">
+         <property name="contextMenuPolicy">
+          <enum>Qt::NoContextMenu</enum>
          </property>
          <property name="title">
-          <string>Language &amp;&amp; data encoding</string>
+          <string>Locale and language</string>
          </property>
-         <layout class="QGridLayout" name="gridLayout_groupBox_encoding">
-          <item row="0" column="0">
+         <layout class="QGridLayout" name="gridLayout_groupBox_locale">
+          <item row="0" column="0" rowspan="2">
+           <widget class="QToolButton" name="toolButton_holder_for_language_icon">
+            <property name="enabled">
+             <bool>false</bool>
+            </property>
+            <property name="cursor">
+             <cursorShape>ForbiddenCursor</cursorShape>
+            </property>
+            <property name="text">
+             <string notr="true"/>
+            </property>
+            <property name="icon">
+             <iconset resource="../mudlet.qrc">
+              <normaloff>:/icons/language-icon-256x.png</normaloff>
```

Comment:
This **is** important I feel because if the UI has come up in a language which you, as a newbie user, cannot read - this icon is increasingly recognised to mean this is where you can change it to **your** language. Remember the user will probably not see the text in their language until they click on the `QComboBox` to get it to drop down and show more/all (there is a separate issue about that IIRC) of the entries so they can then choose it.

*That actually points out that the message that says you should restart Mudlet for the change to take effect is not any good if you do not understand the currently used language - though it might now be possible to hook into that language change signal that I've added - so that the text shown can be redrawn in the selected language even if everything else has to wait for a restart....*

Reply:
I don't agree, sorry. It is not internationally recognised, and there are better ways to fixing the problem of an unknown language for the user - such as autodetecting it from the desktop. PR also isn't aiming to solve this problem (title is "Get Qt translations working"). We're back to PRs that leak in scope...

Meanwhile, everyone who's language works (majority of the userbase) has to pay the price of something unaesthetic when it's not necessary in practice. Not good to add small things for small edgecases so in the end everything is ugly for everyone and everyone hates it :(

Reply:
Okay, removed in next commit.